.travis.yml

```diff
@@ -26,14 +23,19 @@ env:
     - secure: yIrHdkLhWtO37N24cRU71PK80zvkPzlJ1Jm+1MS3C6/icgJ8At1N/UgpBdxuBs1tlJm1O7Ahk+/MhpGWlspx9tCF7Hzw9Rj0xpKhqsvM155zgH582xNPwNebiCr/tmsH2/ELgFBBgC9sCW5luzeKx9nyuedGA4fmD2fK5Phc5EE=
     - secure: fGRo+RTDcnWNnYjSgDtHIRFrkXDxYzFrF0JaGs73ZunLXoYbJ5FclaM9+wjw8BbGEEsVbyBYMSHyy8E2Y0VRggal/XlFkKaqQQHSDVuslcM5YdAkcLC8jFSuSzP32X6cY+L9fKszFVaO+vn84lXwiQnqgpOHAtN32/eB//5Ky1s=
     - secure: m9ylFoxkeoOjBTYi7IF93Rqq95uJkL95onYJc5k1Xo2wNCDrAfvZ8cv6o70Qr/lpWqvgxPueDP6VXjxKAmSueQnx9uGKyigFCpMH4zl7+7n7l4382Eeyex2uE6kCh/lA4EjkmE3QdIW6l8YCGJD1IoHqVSKPZ67ZSdnapTgGPYQ=
-  matrix:
-    - BROWSER=chrome_headless ARTIFACTS_BUCKET=recurly-js REPORT_COVERAGE=true
-    - BROWSER=chrome
-    - BROWSER=firefox
-    - BROWSER=safari
-    - BROWSER=edge
-    - BROWSER=ie_11
-    - BROWSER=ios_12
-    - BROWSER=android_9
-    - BROWSER=android_8
-    - BROWSER=android_7
+jobs:
+  include:
+    - stage: Lint
+      script: make lint
+    - stage: Test
+      env: BROWSER=chrome_headless ARTIFACTS_BUCKET=recurly-js REPORT_COVERAGE=true
+      script: make test-ci
+    - env: BROWSER=chrome
+    - env: BROWSER=firefox
+    - env: BROWSER=safari
+    - env: BROWSER=edge
+    - env: BROWSER=ie_11
+    - env: BROWSER=ios_12
+    - env: BROWSER=android_9
+    - env: BROWSER=android_8
+    - env: BROWSER=android_7
```

Comment:
I changed the matrix spec here so that the linting process, if it fails, will fail the build before proceeding to run the selenium grid